mapApp/templates/mapApp/navbar.html

```diff
@@ -56,6 +56,13 @@
        {% endif %}
         <li>{% blocktrans %}<a href="https://bikemaps.org/fr">Français</a>{% endblocktrans %}</li>
         <li>{% blocktrans %}<a href="https://bikemaps.org/is">Íslenska</a>{% endblocktrans %}</li>
+        {% if LANGUAGE_CODE == 'fr' %}
+          <li><a href="https://onmarcheonroule.org/" class="navbar-icon" style="height:35px; margin-right:0px"><img src="{% static 'mapApp/images/WalkRollMap_FRE_RGB_300dpi.png' %}"></a></li>
+        {% else %}
+          <li><a href="https://walkrollmap.org/" class="navbar-icon" style="height:35px; margin-right:0px"><img src="{% static 'mapApp/images/WalkRollMap_ENG_RGB_300dpi.png' %}"></a></li>
```

Comment:
Do you have access to a WalkRollMap icon without the text? The text is so small it can't be read, so I was thinking it might look better if you used an icon without any text. What do you think?